src/containers/ConcentricCircles/EdgeLayout.js

```diff
@@ -15,24 +13,11 @@ export class EdgeLayout extends PureComponent {
     displayEdges: [],
   };
 
-  renderEdge = ({ key, from, to }) => {
-    if (!from || !to) { return null; }
+  renderEdge = edge => (<Edge {...edge} />);
```

Comment:
I'd be inclined to make `key` explicit, since it's this component (`EdgeLayout`) that makes it a requirement.